x-pack/plugins/siem/public/components/stat_items/index.tsx

```diff
@@ -177,13 +207,13 @@ export const StatItemsComponent = React.memo<StatItemsProps>(
           <EuiFlexGroup>
             {enableBarChart && (
               <FlexItem>
-                <BarChart barChart={barChart!} />
+                <BarChart barChart={barChart!} configs={barchartConfigs} />
               </FlexItem>
             )}
 
             {enableAreaChart && (
               <FlexItem>
-                <AreaChart areaChart={areaChart!} />
+                <AreaChart areaChart={areaChart!} configs={areachartConfigs} />
```

Comment:
You don't need to turn off the type assertion checking. `{barChart!}`, can be just `{barChart}`.